src/main/java/com/ekart/thunderbolt/controller/ProductController.java

```diff
@@ -0,0 +1,126 @@
+package com.ekart.thunderbolt.controller;
+
+
+import com.ekart.thunderbolt.DAO.ProductDAO;
+import com.ekart.thunderbolt.service.ProductService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.AllArgsConstructor;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import static org.springframework.http.ResponseEntity.status;
+
+@RestController
+@RequestMapping("/api/product")
+@AllArgsConstructor
+@Tag(name="Product" , description = "Product APIs")
+public class ProductController {
+
+    private final Logger LOGGER = LoggerFactory.getLogger(ProductController.class);
+    private final ProductService productsService;
```

Comment:
Doesn't this need to be Autowired? Verify and update if necessary.

Reply:
@AllArgsContructor annotation in Lombok takes care of Auto Wiring.